api.js

```diff
@@ -653,12 +654,18 @@ API.prototype.getBrowserTimingHeader = function getBrowserTimingHeader(options)
   const tabs = config.browser_monitoring.debug ? 2 : 0
   const json = JSON.stringify(rumHash, null, tabs)
 
-  // set nonce attribute if passed in options
-  const nonce = options && options.nonce ? 'nonce="' + options.nonce + '"' : ''
-  const script = options && options.hasToRemoveScriptWrapper ? RUM_STUB : RUM_STUB_SHELL
-
   // the complete header to be written to the browser
-  const out = util.format(script, nonce, json, jsAgentLoader)
+  const out =
```

Comment:
it took me a bit to realize that `nonce` and `hasToRemoveScriptWrapper` are mutually exclusive.  nice catch!